### ISSUE TITLE
document/publish schema/spec

### ISSUE THREAD

**Issue by dseynaev** [r-hub/pkgsearch#116]
Hi! Thanks for this package and the work on crandb

Is there a formal schema/spec somewhere of the resources returned by https://crandb.r-pkg.org/ ?

**Comment by JosiahParry** [comment]
Agreed! Is there a public api that we can query outside of pkgsearch?

**Comment by gaborcsardi** [comment]
No, there isn't, sorry.
